packages/rrweb/test/integration.test.ts

```diff
@@ -495,18 +496,21 @@ describe('record integration tests', function (this: ISuite) {
     assertSnapshot(snapshots);
   });
 
-  it('should mask value attribute with maskInputOptions', async () => {
+  it('should always mask value attribute of passwords', async () => {
     const page: puppeteer.Page = await browser.newPage();
     await page.goto('about:blank');
     await page.setContent(
       getHtml.call(this, 'password.html', {
-        maskInputOptions: {
-          password: true,
-        },
+        maskInputOptions: {},
       }),
     );
 
-    await page.type('input[type="password"]', 'secr3t');
+    await page.type('#password', 'secr3t');
+
+    // Change type to text (simulate "show password")
+    await page.click('#show-password');
+    await page.type('#password', 'XY');
+    await page.click('#show-password');
```

Comment:
This toggles it back to `type: password`? It might be better to test leaving it as text

Reply:
We test before this line (on 512) that it works while being a text field, so I think this should be OK?